assets/schema_input.json

```diff
@@ -10,14 +10,19 @@
             "sample": {
                 "pattern": "^\\S+$",
                 "errorMessage": "Sample name must be provided and cannot contain spaces",
+                "meta": ["irida_id"]
+            },
+            "sample_name": {
+                "pattern": "^\\S+$",
```

Comment:
Could you remove the `pattern` here?

IRIDA Next allows whitespace in sample names. Rather than restricting those characters here (which would cause the pipeline to fail), we have been having no `pattern` to restrict sample names in the JSON schema, and instead replacing restricted characters in the workflow code with underscores `_`.